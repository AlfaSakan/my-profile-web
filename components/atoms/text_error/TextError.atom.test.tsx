import renderer from 'react-test-renderer';
import TextError from './TextError.atom';

describe('TextError component test', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<TextError errorMessage="error" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
