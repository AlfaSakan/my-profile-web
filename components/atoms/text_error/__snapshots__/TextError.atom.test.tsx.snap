// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextError component test should render correctly 1`] = `
<div
  className="flex items-center absolute mt-1"
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root"
    focusable="false"
    style={
      Object {
        "fill": "rgb(239 68 68)",
        "fontSize": 14,
        "marginRight": 4,
      }
    }
    viewBox="0 0 24 24"
  >
    <path
      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
    />
  </svg>
  <p
    className="text-red-500 text-xs"
  >
    error
  </p>
</div>
`;
